test/JDBC/input/views/sys-all_columns.sql

```diff
@@ -36,3 +36,35 @@ GO
 DROP TABLE IF EXISTS sys_all_columns_table
 DROP TABLE IF EXISTS columns_scale_precision_length_test
 GO
+
+DROP TABLE IF EXISTS sys_all_columns_table_one
+GO
+
+CREATE TABLE sys_all_columns_table_one (
+	col_one INT PRIMARY KEY,
+	col_two INT,
+	col_three INT IDENTITY(1,1),
+	col_computed AS col_one * col_two
+)
+GO
+
+SELECT is_computed FROM sys.all_columns WHERE name = 'col_computed'
+GO
+
+SELECT is_computed FROM sys.all_columns WHERE name = 'col_two'
+GO
+
+SELECT is_computed FROM sys.all_columns WHERE name = 'col_one'
+GO
+
+SELECT is_computed FROM sys.all_columns WHERE name = 'random_col'
+GO
+
+SELECT is_identity FROM sys.all_columns WHERE name = 'col_three'
+GO
+
+SELECT is_identity FROM sys.all_columns WHERE name = 'col_two'
+GO
+
+DROP TABLE IF EXISTS sys_all_columns_table_one
+GO
```

Comment:
Please move the tests to sys-all_columns-vu* to test this under both normal and upgrade scenario.
Besides, I think sys-all_columns-dep should be added to latest version's upgrade schedule.

Reply:
Resolved the comments